blackboxopt/visualizations/visualizer.py

```diff
@@ -195,6 +196,117 @@ def multi_objective_visualization(
     return fig
 
 
+def parallel_coordinate_plot_parameters(
+    evaluations: List[Evaluation],
+    columns: Optional[List[str]] = None,
+    color_by: Optional[str] = None,
+):
+    """Create an interactive parallel coordinate plot.
+
+    Useful to investigate relationships in a higher dimensional search space and the
+        optimization's objective(s).
+
+    Args:
+        evaluations: Evaluations to plot.
+        columns: Names of columns to show. Can contain parameter names, objective names
+            and settings keys. If `None`, all parameters, objectives and settings are
+            displayed.
+        color_by: Parameter name, objective name, settings key. The corresponding column
+            will be shown at the very right, it's value will be used for the color
+            scale. If `None`, all lines have the same color.
+
+    Returns:
+        Plotly figure
+
+    Raised:
+        NoSuccessfulEvaluationsError: In case `evaluations` does not contain at least
+            one successful evaluation (an evaluation with objective value != `None`).
+    """
+    if not evaluations:
+        raise NoSuccessfulEvaluationsError
+
+    # Prepare dataframe for visualization
+    df = evaluations_to_df(evaluations)
+
+    # Drop unused columns and indices
+    df = df[["configuration", "settings", "objectives"]]
+    objective_cols = df["objectives"].columns.to_list()
+    settings_cols = df["settings"].columns.to_list()
+    df = df.droplevel(0, axis=1)
+
+    # If no columns are specified, use all:
+    if not columns:
+        columns = df.columns.to_list()
+
+    # Prepare a coordinate (vertical line) for every column
+    coordinates = []
+    colored_coordinate = {}
+    for column in columns:
+        coordinate: Dict[str, Any] = {}
+
+        if column in objective_cols:
+            coordinate["label"] = f"<b>Objective: {column}</b>"
+        elif column in settings_cols:
+            coordinate["label"] = f"Setting: {column}"
+        else:
+            coordinate["label"] = column
+
+        parameter_type = df[column].dtype.name
+        if parameter_type.startswith("float") or parameter_type.startswith("int"):
+            # Handling floats and integers as same, vecause unfortunately it's hard to
+            # use integers only for ticks and still be robust regarding a large range
+            # of values.
+            coordinate["values"] = df[column]
+        elif parameter_type in ["object", "bool"]:
+            # Encode categorical values to integers. Unfortunately, ordinal parameters
+            # loose there ordering, as there is no information about the order in the
```

Comment:
If we want to keep the ordering, we would need to also have the search space here. We can add this later, if we feel that this becomes more important.

Reply:
+1 for doing that later when we needed